### PR TITLE
Mushroom Classification using Deep Neural Networks

### PR DESCRIPTION
**Pull request by PiyushBL45t**
# Pull Request for DL-Simplified 💡

## Issue Title : To classify mushroom breeds using deep learning techniques.

- **Info about the related issue (Aim of the project)** : Perform Image Classification using Deep Neural Networks on different mushroom breeds.
- **Name:** Piyush Bhujbal
- **GitHub ID:** https://ww.github.com/PiyushBL45t
- **Email ID:** bhujbalpiyush@gmail.com
- **Idenitfy yourself: (Mention in which program you are contributing in. Eg. For a JWOC 2022 participant it's, `JWOC Participant`)** Social Summer of Code 2.0 

Closes: #319 

### Describe the add-ons or changes you've made 📃
1. Dataset images were not properly shaped, so wrote a script for resizing the images to a reliable resolution in 256x256.
2. Trained the CNN model using two frameworks **Tensorflow, Pytorch** and tested their accuracy and loss parameters.
3. Created image data for training, testing and validation using Pytorch and tensorflows image data generators.
4. Also used pretrained models like  **InceptionV3, EfficientNet50** to perform testing and predicitons,

## Type of change ☑️

What sort of change have you made:

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Code style update (formatting, local variables)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

## How Has This Been Tested? ⚙️

1. The models trained are tested using images that were converted to Numpy arrays using Keras API.
2. After model training, accuracy and loss parameters were saved and plotted using Pandas.
3. Provided the true versus predicted labels with images using Matplotlib.

## Checklist: ☑️

- [X] My code follows the guidelines of this project.
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly wherever it was hard to understand.
- [X] I have made corresponding changes to the documentation.
- [X] My changes generate no new warnings.
- [X] I have added things that prove my fix is effective or that my feature works.
- [X] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by abhisheks008** [comment]
Can you please rename the notebooks used here in this project.
